tests/data_service.py

```diff
@@ -1,7 +1,7 @@
 import requests
 
 # Endpoint URLs
-NUMIA_API_URL = 'https://data.numia-stage.osmosis.zone'
+NUMIA_API_URL = 'https://stage-proxy-data-api.osmosis-labs.workers.dev'
```

Comment:
Note to reviewer: drive-by fix to integration test flakiness. Numia stage is for some reason broken. Updaed to prod